package.json

```diff
@@ -7,7 +7,8 @@
   "license": "Apache-2.0",
   "scripts": {
     "clean": "rm -rf node_modules/ && npm cache clean --force",
-    "update-lock": "npm run clean && npm i --no-progress",
+    "update-dependencies": "npm run clean && npm i --no-progress",
```

Comment:
That will update the `package-lock.json` no?
So it is not a reproducible build on CI

Reply:
That won't update it as long as we keep it up-to-date. Sadly, there's no other way to do it in CI, since we are using dev dependencies to build test assets. The only way would be having locked versions in our package.json, and that would lead to a PITA when we want to upgrade.

Reply:
Well the CI will update it right? The cache can be invalidated for many reasons, so it will update the package-lock.json

Hopefully peer dependencies will not mess up :)

Reply:
yeah, as I said, the only way to do it for our CI, is locking the version of the dependencies in package.json, since npm doesn't provide a method to install the exact same tree described in the lock (ci is lacking the dev dependencies)